content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,140 +9,249 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
         <th>Maintainer</th>
-        <th>Platform</th>
-        <th>Version</th>
-        <th>Docs</th>
-        <th>Website</th>
+        <th>Target Platform</th>
+        <th>Link</th>
+        <th>Latest Release</th>
       </tr>
     </thead>
     <tbody>
       <tr>
         <td>Kubeflow on AWS</td>
-        <td>Amazon Web Services (AWS)</td>
-        <td>Amazon Elastic Kubernetes Service (EKS)</td>
-        <td>{{% aws/latest-version %}}</td>
-        <td><a href="/docs/distributions/aws/">Docs</a></td>
-        <td><a href="https://awslabs.github.io/kubeflow-manifests">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Kubeflow on Azure</td>
-        <td>Microsoft Azure</td>
-        <td>Azure Kubernetes Service (AKS)</td>
-        <td>1.2</td>
-        <td><a href="/docs/distributions/azure/">Docs</a></td>
-        <td></td>
+        <td>Amazon Web Services</td>
+        <td>
+          Amazon Elastic Kubernetes Service (EKS)
+        </td>
+        <td>
+          <u><a href="https://awslabs.github.io/kubeflow-manifests">Website</a></u>
+        </td>
+        <td>
+          2022-10-14
+          <br>
+          <sup>[<a href="https://github.com/awslabs/kubeflow-manifests/releases">Release History</a>]</sup>
+        </td>
       </tr>
       <tr>
         <td>Kubeflow on Google Cloud</td>
         <td>Google Cloud</td>
-        <td>Google Kubernetes Engine (GKE)</td>
-        <td>{{% gke/latest-version %}}</td>
-        <td><a href="/docs/distributions/gke/">Docs</a></td>
-        <td></td>
+        <td>
+          Google Kubernetes Engine (GKE)
+        </td>
+        <td>
+          <u><a href="https://github.com/GoogleCloudPlatform/kubeflow-distribution">Website</a></u>
+          <br>
+          <sub><u><a href="/docs/distributions/gke/">Legacy Docs</a></u></sub>
+        </td>
+        <td>
+          2022-10-17
+          <br>
+          <sup>[<a href="https://github.com/GoogleCloudPlatform/kubeflow-distribution/releases">Release History</a>]</sup>
+        </td>
       </tr>
       <tr>
         <td>Kubeflow on IBM Cloud</td>
         <td>IBM Cloud</td>
-        <td>IBM Cloud Kubernetes Service (IKS) </td>
-        <td>{{% iks/latest-version %}}</td>
-        <td><a href="/docs/distributions/ibm/">Docs</a></td>
-        <td><a href="https://github.com/IBM/manifests/tree/{{% iks/latest-branch %}}">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Kubeflow on Nutanix</td>
-        <td>Nutanix</td>
-        <td>Nutanix Karbon</td>
-        <td>{{% nutanix/latest-version %}}</td>
-        <td><a href="/docs/distributions/nutanix/">Docs</a></td>
-        <td></td>
+        <td>
+          IBM Cloud Kubernetes Service (IKS)
+        </td>
+        <td>
+          <u><a href="https://github.com/IBM/manifests/tree/{{% iks/latest-branch %}}">Website</a></u>
+          <br>
+          <sub><u><a href="/docs/distributions/ibm/">Legacy Docs</a></u></sub>
+        </td>
+        <td>
+          Unknown
```

Comment:
@yhwang is there something we can reference here to add the date of your latest release?

Reply:
thanks for bringing this to my attention. the latest release date is `2022-10-10` for IKS

Reply:
> thanks for bringing this to my attention. the latest release date is `2022-10-10` for IKS

@yhwang can you share a public announcement, GitHub release, or changelog that has the release date on it? We need a reference so we can link it.

Reply:
https://github.com/IBM/manifests/releases/tag/v1.6.1

Reply:
@yhwang I have updated the table to reflect this information in https://github.com/kubeflow/website/pull/3381/commits/98d74badfafe3c35ddb56ec3d408129d29ff3cdf

Reply:
thanks